src/docs/truffle/getting-started/debugging-your-contracts.md

```diff
@@ -96,6 +104,18 @@ $ truffle debug
 
 Regardless of how you start the debugger, once it is running you are not limited to debugging only the transaction you launched it with; it is possible to unload the current transaction and load a new one, as described below.
 
+You can specify the network you want to debug on with the `--network` option:
+
+```shell
+$ truffle debug <transaction hash> --network <network>
+```
+
+And with the `--fetch-external` option ([see below](#debugging-external-contracts-with-verified-source)), you can debug contract instances outside your project that have verified source code on Etherscan.  (Support for retrieving code from Sourcify as well is planned.)  When using this option, you must specify a transaction hash to debug, and you will not be able to switch transactions from inside the debugger.
```

Comment:
Probably needs link to Sourcify (and for consistency, also Etherscan)

Reply:
Oh, good point.